resources/assets/uad_lists.json

```
@@ -36598,5 +36597,572 @@
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.huawei.calculator",
+    "list": "Oem",
+    "description": "Calculator app",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.huawei.coauthservice",
+    "list": "Oem",
+    "description": "HwCoAuthService\nApplock, lockscreen password, face recognition things, fingerprint, not sure if it's worth removing it",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  {
+    "id": "com.huawei.contentsensor",
+    "list": "Oem",
+    "description": "ContentSensor\nLooks like this app has a lot chinese code and collects a lot stuff about user like apps installed, app info, connects to huawei hivoice sites",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.huawei.controlcenter",
+    "list": "Oem",
+    "description": "Smart Collaboration\nCollaboration between devices. Needed for Multi-Screen, Wireless Projection.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.huawei.def",
+    "list": "Oem",
+    "description": "DEF\nUnused? message things:allows the app public keys on connected devices, getting status, access information",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.huawei.deviceauth",
+    "list": "Oem",
+    "description": "TrustedDeviceAuth\nI guess it's for authentication keys",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.huawei.devicegroupmanage",
+    "list": "Oem",
+    "description": "(Unused?)HwGroupManager\nIt's an app without activities, only device info things",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  {
+    "id": "com.huawei.devicemanager",
+    "list": "Oem",
+    "description": "Needed for Huawei account. I found only things: installing service?, Linked devices, Multi-Device management, connection code, metrics",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.huawei.distributed.kms",
+    "list": "Oem",
+    "description": "HwDistributedKeyManager\nHas random frameworks, probably needed for huawei account, authentication",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.huawei.dmsdp",
+    "list": "Oem",
+    "description": "Multi-Device management\nThis service allows you to virtually expand your devices capabilities\nby connecting to other devices and using them as an extension of this device.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Advanced"
```
Some users may want to use this.